bundles/org.openhab.binding.lutron/src/main/resources/ESH-INF/thing/thing-types.xml

```diff
@@ -94,7 +94,8 @@
 				<label>Turn On To Last Level</label>
 				<description>
 					If set to true, dimmer will go to the last non-zero level set when an ON command is received.
-					If the last level cannot be determined, the value of onLevel will be used instead.
+					If the
+					last level cannot be determined, the value of onLevel will be used instead.
```

Comment:
I don't understand why it made this change, The line isn't as long as the one above it, and the break was inserted in a very odd place.

Reply:
Yes I also noticed that and have now created #8413 to improve the formatting. It seems to format well when all text is on one line but seems to bug when there is still room left on previous lines.